server/routes/index.js

```diff
@@ -366,17 +366,17 @@ module.exports = function Index ({ authenticationMiddleware }) {
       data: response.cases
     }
     session.confirmedMatch = undefined
-    session.matchName = undefined
     session.matchType = 'bulk'
     session.matchDate = date
     session.courtCode = courtCode
     res.render('match-records', templateValues)
   })
 
-  router.get('/:courtCode/case/:caseId/match/defendant/:defendantId', defaults, async (req, res) => {
-    const { params: { caseId, defendantId }, session, path } = req
+  router.get('/:courtCode/case/:caseId/hearing/:hearingId/match/defendant/:defendantId', defaults, async (req, res) => {
```

Comment:
Is it correct that the matching routes still have `/case/:caseId` ?

Reply:
Yes, this is to retain both caseId and hearingId for further processing which otherwise would need storing one of them in the session